streamlit_web.py

```diff
@@ -140,7 +146,7 @@ def retreive_data(data_retreive):
         except TypeError:
             st.sidebar.warning(
                 "No data imported. Please check the reflection document input")
-            landing_src()
+            readme()
```

Comment:
Any reason why this is changed? I think this was changed to `landing_src()` by a previous PR.

Reply:
It looks like that just merged strangely, it's updated back to landing_src() now